### PR TITLE
Misleading documentation of "either" and "both" methods fix

### PR DESCRIPTION
**Pull request by karatsuba**
Hello! Thanks a lot for such a cool library! I really like it!

Here is a fix for https://github.com/ramda/ramda/issues/2244

### CODE REVIEW

File: source/both.js [hunk 1]
```diff
@@ -31,6 +33,9 @@ import lift from './lift';
  *      var f = R.both(gt10, lt20);
  *      f(15); //=> true
  *      f(30); //=> false
+ *
+ *      both(Maybe.Just(false), Maybe.Just(55)); // => Maybe.Just(false)
```
Comment (kedashoe) [inline source/both.js]: `both` -> `R.both` (it is standard for the doc examples to use the `R.` prefix)

File: source/both.js [hunk 2]
```diff
@@ -12,6 +12,8 @@ import lift from './lift';
  * meaning that the second function will not be invoked if the first returns a
  * false-y value.
  *
+ * Also the method returns not only a function, but any applicative functor of corresponded type.
```
Comment (kedashoe) [inline source/both.js]: I don't think we need this, there is already a note about these functions working with any applicative

**Comment by kedashoe** [comment]
ping @karatsuba

**Comment by karatsuba** [comment]
hi @kedashoe! I pushed my changes 😄

**Pull request merged**